app/jobs/in_person/send_proofing_notification_job.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module InPerson
+  class SendProofingNotificationJob < ApplicationJob
+    # @param [Number] enrollment_id primary key of the enrollment
+    def perform(enrollment_id)
+      return unless IdentityConfig.store.in_person_proofing_enabled &&
+                    IdentityConfig.store.in_person_send_proofing_notifications_enabled
+      begin
+        enrollment = InPersonEnrollment.find_by(
+          { id: enrollment_id },
+          include: [:notification_phone_configuration, :user],
+        )
+        return unless enrollment
+        # skip when enrollment status not success/failed/expired and no phone configured
+        if enrollment.skip_notification_sent_at_set?
+          # log event
+          analytics(user: enrollment.user).
+            idv_in_person_usps_proofing_results_notification_job_skipped(
+              enrollment_code: enrollment.enrollment_code,
+              enrollment_id: enrollment.id,
+            )
+          return
+        end
+        analytics(user: enrollment.user).
+          idv_in_person_usps_proofing_results_notification_job_started(
+            enrollment_code: enrollment.enrollment_code,
+            enrollment_id: enrollment.id,
+          )
+        if enrollment.expired?
+          # no sending message for expired status
+          enrollment.notification_phone_configuration&.destroy
+          return
+        end
```

Comment:
Since we are now deleting expired enrollments' phone numbers in the proofing results job, this conditional is redundant.

Reply:
@NavaTim , i thought about it, since there may be a small chance to run the job manually with specific enrollment, i kept it here.